offload/CMakeLists.txt

```diff
@@ -159,13 +168,20 @@ set (LIBOMPTARGET_TESTED_PLUGINS "")
 string( TOLOWER "${CMAKE_BUILD_TYPE}" LIBOMPTARGET_CMAKE_BUILD_TYPE)
 if(LIBOMPTARGET_CMAKE_BUILD_TYPE MATCHES debug)
   option(LIBOMPTARGET_ENABLE_DEBUG "Allow debug output with the environment variable LIBOMPTARGET_DEBUG=1" ON)
+  add_definitions(-DDEBUG)
```

Comment:
What's this for? I don't think we want a global definition `DEBUG` set for every compilation.